taui/src/components/neighborhood-details.js

```diff
@@ -190,7 +190,7 @@ export default class NeighborhoodDetails extends PureComponent<Props> {
     // Look up the currently selected user profile destination from the origin
     const originLabel = origin ? origin.label || '' : ''
     const currentDestination = userProfile.destinations.find(d => d.location.label === originLabel)
-    const { id, town, wikipedia } = neighborhood.properties
+    const { id, town, town_website_description } = neighborhood.properties
```

Comment:
This is making the linter grumpy, it probably needs to be switched off here and below.